Clean up Readme (#3084)
DataDog/integrations-core: openstack_controller/README.md

### Configuration

#### Prepare OpenStack

> Create a datadog user that will be specified in your `openstack_controller.d/conf.yaml` file. This user requires admin read only permissions across your environment so that it can be run from a single node and read high level system information about all compute nodes and servers. 

Reviewer: ```suggestion
Create a `datadog` user this is used in your `openstack_controller.d/conf.yaml` file. This user requires admin read only permissions across your environment so that it can be run from a single node and read high level system information about all compute nodes and servers. 
```

Reviewer: ^ Remove future tense from docs.

Author: Done, thanks 🙇
